egui/src/widgets/plot/items.rs

```diff
@@ -34,13 +34,151 @@ impl Value {
 
 // ----------------------------------------------------------------------------
 
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum LineStyle {
+    Solid,
+    Dotted { px: f32 },
+    Dashed { px: f32 },
```

Comment:
"px" is a weird name. Call it "radius" and/or "length" if that is what it is.

Reply:
Changed in 80936ba